scripts/schemas/board-schema.yml

```diff
@@ -78,3 +78,56 @@ mapping:
     type: seq
     sequence:
       - include: board-schema
+  run_once:
+    type: seq
+    desc: |
+          Allows for restricting west flash commands when using sysbuild to run once per given
+          grouping of boards. This is to allow for future image program cycles to not erase the
+          flash of a device which has just been programmed by another image.
+    sequence:
+      - type: map
+        mapping:
+          regex;(.*):
+            required: true
+            type: seq
+            desc: |
+                  A list of commands which should be limited to running once per invocation of
+                  west flash for a set of boards
+            sequence:
+              - type: map
+                mapping:
+                  boards:
```

Comment:
Why is there a list `boards` with a list of board names ?
What I refer to is, why this:
```
<key>:
- boards:
  - foo
  - bar
- boards:
  - 
```
Instead of:
```
command: <value>
boards:
- foo
- bar
- ...
```